wp-content/themes/csisnuclearnetwork/assets/_scss/pages/single.scss

```diff
@@ -243,24 +240,33 @@
     &-heading {
       display: flex;
       gap: rem(12);
-      justify-content: center;
+      justify-content: start;
       align-items: center;
       margin-bottom: rem(24);
-      @extend %text-style-heading-large-strong-caps;
+      @extend %text-style-heading-large-strong;
```

Comment:
You'll need to do the typography for this heading without using a placeholder, since it doesn't match any in this project.  You'll also need to use a separate class, since this class effects all headings in the post footer (ie Authors).

Reply:
i think i undid this, but there might be more places where i need to revert my previous changes